manim/animation/animation.py

```diff
@@ -156,8 +156,11 @@ def __init__(
                 mobject if mobject is not None else OpenGLMobject()
             )
         else:
-            self.starting_mobject: Mobject = Mobject()
-            self.mobject: Mobject = mobject if mobject is not None else Mobject()
+            self.starting_mobject: Mobject = Mobject(name=self.__class__.__name__)
+            if mobject:
+                self.mobject: Mobject = mobject
+            else:
+                self.mobject: Mobject = self.starting_mobject
```

Comment:
**Nitpick** : This would be better in one line :)

Reply:
Done.